### ISSUE TITLE
How to replace `io.load_txt` ?

### ISSUE THREAD

**Issue by shiba24** [alexlib/pivpy#43]
Hi,
I noticed from v0.0.16 you deleted `io.load_txt` function.
I could use 0.0.15 but could you tell me how I can replace it? For example, in your jupyter notebook: https://github.com/alexlib/openpiv_bos_velocimetry/blob/master/1_Example_BOS_PIV.ipynb

Thank you!

**Comment by alexlib** [comment]
Hi @shiba24 
I guess you're looking for `io.load_openpiv_txt` https://github.com/alexlib/pivpy/blob/1df1d8b3567e91026f55d72efc84e6854cf007d9/pivpy/io.py#L519

i found that also old Davis was .txt and it's now called load_davis8_txt https://github.com/alexlib/pivpy/blob/1df1d8b3567e91026f55d72efc84e6854cf007d9/pivpy/io.py#L584

**Comment by shiba24** [comment]
Thank you @alexlib, I checked both and it seems that the results are different between `io.load_txt` and `io.load_openpiv_txt`. For now, I will keep using 0.0.15.

**Comment by alexlib** [comment]
Please upload here the file you read and explain the difference. 

Thanks

**Comment by alexlib** [comment]
@shiba24, you're right. I see the problem - the order of the x,y is scrambled. I will try to fix it soon.

**Comment by alexlib** [comment]
I think I fixed it in 0.0.18 version, but I keep the name of `load_openpiv_txt`
